pkg/apis/hive/v1/zz_generated.deepcopy.go

```diff
@@ -1791,6 +1792,11 @@ func (in *Platform) DeepCopyInto(out *Platform) {
 		*out = new(gcp.Platform)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.BareMetal != nil {
+		in, out := &in.BareMetal, &out.BareMetal
+		*out = new(baremetal.Platform)
+		**out = **in
```

Comment:
Just a question here, why not `DeepCopyInto` like other cloud providers?

Reply:
This is generated code, I honestly am not sure why it's doing that differently, possibly because it's just an empty struct today.